systemd/sm-mpath-root.service

```diff
@@ -1,13 +1,10 @@
 [Unit]
 Description=Ensure symlinks for multipathed rootdev
-Wants=basic.target multipathd.service
-After=basic.target
-Before=multipathd.service
 
 [Service]
 Type=oneshot
 RemainAfterExit=yes
 ExecStart=/opt/xensource/sm/multipath-root-setup
 
 [Install]
-WantedBy=multi-user.target
+WantedBy=sysinit.target
```

Comment:
According to ```man bootup``` on my system, ```sysinit.target``` occurs strictly before ```basic.target```, and according to ```man systemd.special```

```
       basic.target
           A special target unit covering basic boot-up.

           systemd automatically adds dependency of the type After= for this target unit to all services (except for those with DefaultDependencies=no).
```

So do we need to put ```DefaultDependencies=no``` in here?

Reply:
it doesn't seem so. But we should probably perform a separate system wide check of the startup sequencing before GA.